install_config/install/deploy_router.adoc

```diff
@@ -609,6 +609,58 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ====
 
 
+=== Preventing Connection Failures During Restarts ===
+
+If you connect to the router while the proxy is reloading, there is a
+small chance that your connection will end up in the wrong network
+queue and be dropped.  There is kernel work going on to remove that
+problem, but in the meantime it is possible to work around the problem
+by installing *_iptables_* rules to prevent connections during the
+reload window.  However, doing so means that the router needs to run
+with elevated privilege so that it can manipulate *_iptables_* on the
+host.  It also means that connections that happen during the reload
+are temporarily ignored and will have to retransmit their connection
+start, lengthening the time it takes to connect, but preventing
+connection failure.
+
+Because of these reasons, and because the risk is small, we do not
```

Comment:
Consider replacing this sentence with:
Due to these issues, do not enable this option most of the time.